File: app/src/main/java/com/bytedance/tiktok/base/BaseRvViewHolder.java
package com.bytedance.tiktok.base;

import android.view.View;
import androidx.recyclerview.widget.RecyclerView;
import butterknife.ButterKnife;

/**
 * create by libo
 * create on 2018/11/15
 * description Recycler ViewHolder基类
 */
public class BaseRvViewHolder extends RecyclerView.ViewHolder {

    public BaseRvViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/com/bytedance/tiktok/base/BaseRvViewHolder.kt
package com.bytedance.tiktok.base

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import butterknife.ButterKnife

/**
 * create by libo
 * create on 2018/11/15
 * description Recycler ViewHolder基类
 */
open class BaseRvViewHolder(itemView: View?) : RecyclerView.ViewHolder(itemView!!) {
    init {
        ButterKnife.bind(this, itemView!!)
    }
}